### ISSUE TITLE
White page

### ISSUE THREAD

**Issue by dotur911** [FabulosoDev/docker-ps4-pkg-sender#3]
I have set up docker version, all seems fine but i get only white page when i try to acess web ui. How can i fix this?

**Comment by pengytek** [comment]
Did you ever figure this out?

**Comment by dotur911** [comment]
Yes,when you bind shared folder for pkgs ,put some folder inside with some pkg file and it will be shown.
